### ISSUE TITLE
Kodi warns about old settings being used

### ISSUE THREAD

**Issue by linqcan** [kodi-svtplay/xbmc-svtplay#234]
```
DEBUG: CAddonSettings[plugin.video.svtplay]: loading setting definitions
DEBUG: CAddonSettings[plugin.video.svtplay]: trying to load setting definitions from old format...
WARNING: CAddonSettings[plugin.video.svtplay]: cannot reference setting (relative index: -1; absolute index: -1) in another category in old setting condition "eq(-1,false)" for "bwselect"
WARNING: CAddonSettings[plugin.video.svtplay]: failed to parse enable condition "eq(-1,false)" of old setting definition for "bwselect"
WARNING: CAddonSettings[plugin.video.svtplay]: cannot reference setting (relative index: -2; absolute index: -1) in another category in old setting condition " eq(-2,false)" for "bandwidth"
WARNING: CAddonSettings[plugin.video.svtplay]: failed to parse enable condition "eq(-1,true) + eq(-2,false)" of old setting definition for "bandwidth"
DEBUG: CAddonSettings[plugin.video.svtplay]: loading setting values
INFO: CPythonInvoker(6, /storage/.kodi/addons/plugin.video.svtplay/def
```

**Comment by linqcan** [comment]
More warnings:

```
ERROR: NEWADDON Unknown Video Info Key "fanart"
ERROR: NEWADDON Unknown Video Info Key "onlyavailableinsweden"
ERROR: NEWADDON Unknown Video Info Key "poster"
```

**Comment by linqcan** [comment]
Fixed in v5.0.0 (branch krypton).
